src/scenes/Event/store.js

```diff
@@ -97,9 +101,40 @@ export default class EventStore {
 
   @computed get withdrawableAddress() {
     const { currentAddress } = this.app.wallet;
+    const resultIndex = this.event && this.event.currentResultIndex;
+    const profitCut = this.event && this.event.arbitrationRewardPercentage;
+    if (this.betterBets.length === 0 ||
+      this.betterVotes.length === 0 ||
+      this.totalBets.length === 0 ||
+      this.totalVotes.length === 0) return {};
+    const betterBetsSum = sum(this.betterBets);
+    const betterVotesSum = sum(this.betterVotes);
+    const totalBetsSum = sum(this.totalBets);
+    const totalVotesSum = sum(this.totalVotes);
+
+    const totalWinningBets = this.totalBets[resultIndex];
+    const totalLossBets = totalBetsSum - totalWinningBets;
+    const arbitrationShare = (totalLossBets * profitCut) / 100;
+    const betShare = totalLossBets - arbitrationShare;
+    const betterBetsReturn = this.betterBets[resultIndex] + ((betShare * this.betterBets[resultIndex]) / this.totalBets[resultIndex]) || 0;
+
+    const totalWinningVotes = this.totalVotes[resultIndex];
+    const totalLossVotes = totalVotesSum - totalWinningVotes;
+    const totalVotesShare = totalLossVotes + arbitrationShare;
+    const betterVotesReturn = this.betterVotes[resultIndex] + ((totalVotesShare * this.betterVotes[resultIndex]) / this.totalVotes[resultIndex]) || 0;
+    const totalInvestment = betterBetsSum + betterVotesSum + this.escrowAmount;
```

Comment:
technically you shouldn't include escrow amount since you are not getting any increased return on it.

Reply:
TT thinks it should be included since you need to pay them at front